plugins/kubernetes/src/components/Pods/PodsTable.tsx

```diff
@@ -34,74 +27,42 @@ export type PodColumns = 'READY' | 'RESOURCE';
 type PodsTablesProps = {
   pods: V1Pod[];
   extraColumns?: PodColumns[];
+  customColumns?: TableColumn<V1Pod>[];
```

Comment:
Yeah so I'd suggest getting rid of both of these props, and instead just having a single `columns?: TableColumn<V1Pod>[] | undefined` optional prop.